src/tools/clippy/tests/ui/cast.rs

```diff
@@ -17,6 +17,8 @@
     clippy::identity_op
 )]
 
+// FIXME(f16_f128): add tests once const casting is available for these types
```

Comment:
Can those tests be added with #126608 being merged? If so, after merging this PR, please add those in 126608

Reply:
#126608 won't add casting, but it unblocks a followup PR that does add it. I'll make sure to add the Clippy tests with that one.